.github/workflows/spread-tests.yaml

```diff
@@ -25,12 +25,42 @@ on:
         description: 'The rule .yaml file to use (found under tests/lib/spread/rules) for test discovery'
         required: true
         type: string
+      use-snapd-snap-from-master:
+        description: 'If true, will use the snapd snap built on the master branch'
+        required: false
+        type: boolean
+        default: false
+      spread-snapd-deb-from-repo:
+        description: 'If true, will use the snapd package from the repository when possible'
+        required: false
+        type: boolean
+        default: true
+      spread-experimental-features:
+        description: 'Comma-separated list of experimental snapd features to enable with: snap set system "experimental.<feature-name>=true"'
+        required: false
+        type: string
 
 
 jobs:
   run-spread:
+    env:
+      SPREAD_SNAPD_DEB_FROM_REPO: ${{ inputs.spread-snapd-deb-from-repo }}
```

Comment:
We need to be able to SPREAD_SNAP_REEXEC set to 0, otherwise we are re-executing to the snapd snap

Reply:
Correct me if I'm wrong, but I think that should be a future improvement for when/if it is necessary. Currently, the only reason the `SPREAD_SNAPD_DEB_FROM_REPO` is exposed is to be able to set it to false for the `spread-test-build-from-current` job (it's set to default true, just like in `spread.yaml`). From what I just read about snap reexec (which could be wrong), it's only necessary to set it to 0 if you want to manipulate the deb. So with that particular job, it doesn't need to set SNAP_REEXEC to 0. So no job would be actually using that value. Personally, I think it only makes sense to add it when it's needed. If you say that it is needed because the jobs should in fact be setting that value but aren't doing so, that seems like a candidate for a separate PR, since I imagine changing that behavior could break tests, which seems out of the scope of this PR, no?